cmd/draupnir-create-instance

```diff
@@ -16,6 +16,15 @@ if ! [[ "$#" -eq 4 ]]; then
   exit 1
 fi
 
+die_and_stop() {
+  echo "$*" 1>&2
+
+  echo "Stopping instance"
+  sudo -u postgres "$PG_CTL" -w -D "$INSTANCE_PATH" stop
```

Comment:
We should also call `draupnir images destroy ${INSTANCE_ID}` to clean it up from the database and delete the btrfs subvolume. 

The draupnir-destroy-instance script that draupnir calls as part of the destroy also does the same stopping of the database that you have here but I don't think we want to rely on that in case there is a failure somewhere else. It also is safe stop here before hand as the script handles it already having been stopped.